src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -1977,10 +1977,10 @@ public void ReferenceManagerReuse_WithOptions()
             c2 = c1.WithOptions(TestOptions.ReleaseDll);
             Assert.False(c1.ReferenceManagerEquals(c2));
 
-            c2 = c1.WithOptions(new CSharpCompilationOptions(OutputKind.WindowsApplication));
+            c2 = c1.WithOptions(new CSharpCompilationOptions(OutputKind.WindowsApplication)); // Do you want to introduce a new DebugWindowsApplication field in TestOptions?
```

Comment:
Probably not

Reply:
@RikkiGibson Then I should also revert `DebugWindowsRuntimeApp` that I added to TestOptions in this PR right?